src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -178,9 +197,16 @@ public static String detectSystemMypyPath() {
                     .lines()
                     .findFirst();
             process.waitFor();
+            String error = new BufferedReader(new InputStreamReader(process.getErrorStream()))
```

Comment:
**Reporter**: Error Prone
**Rule**: DefaultCharset
**Severity**: WARN
**File**: home/travis/build/leinardi/mypy-pycharm/src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java L200

Implicit use of the platform default charset, which can result in differing behaviour between JVM executions or incorrect behavior if the encoding of the data source doesn&#39;t match expectations.&#10;&#10;String error &#61; new BufferedReader(new InputStreamReader(process.getErrorStream()))^(see http://errorprone.info/bugpattern/DefaultCharset)Did you mean &#39;String error &#61; new BufferedReader(new InputStreamReader(process.getErrorStream(), UTF_8))&#39; or &#39;String error &#61; new BufferedReader(new InputStreamReader(process.getErrorStream(), Charset.defaultCharset()))&#39;?


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-179191721>*